composer.json

```diff
@@ -71,6 +71,9 @@
         ],
         "setup-git-hooks": [
             "cp .git-hooks/* .git/hooks/."
+        ],
+        "reset": [
+            "php artisan db:wipe", "php artisan migrate", "php artisan db:seed"
```

Comment:
Put an `@` in front of the `php` to ensure the proper php is being called instead of some arbitrary php on the machine

https://getcomposer.org/doc/articles/scripts.md#executing-php-scripts